common/content/world_content_service.h

```diff
@@ -23,6 +23,7 @@
 
 #include <string>
 #include <vector>
+#include "../database.h"
```

Comment:
Can forward declare with `class Database;` here instead of pulling in the header dependency in this header to avoid increased incremental build times